ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1049,7 +1047,7 @@
           only_trigger_phrase: true
           trigger_permit_all: true
           status_context: jenkins-multicluster
-          status_url: https://jenkins.antrea-ci.rocks/job/antrea-multicluster-for-pull-request
+          status_url: https://jenkins.antrea.io/job/antrea-multicluster-for-pull-request
```

Comment:
Not sure why this job is special. Other `{name}-{test_name}-for-pull-request` jobs have `status_url: null`.

I can see conformance, e2e, whole-conformance, networkpolicy ... jobs in projects-cloud.yaml. Do they require CAPV to run? I think we don't have CAPV in the new Jenkins.

Reply:
We can support CAPV in new jenkins by running registration command in agent instead of the SSH method.